xml/operations-opsconsole-en-cs_opsconsole_hardware_baremetal.xml

```diff
@@ -36,8 +36,8 @@
   <listitem>
    <para>
     View and provision bare metal servers that are not managed by HPE OneView
-    by installing an HPE Linux or RHEL KVM operating system. You can then
-    activate the servers as compute hosts.
+    by installing an &sle_repo; KVM operating system. You can then activate the
```

Comment:
Hm, does HPE still support RHEL here? If so, you should add that, but vendored with vendor="hpe"